### PR TITLE
PS tarball job updated with LTS releases

### PR DESCRIPTION
**Pull request by kaushikpuneet07**

### CODE REVIEW

**Comment by kaushikpuneet07** [comment]
successful job link with updates : https://ps80.cd.percona.com/job/test-ps-binary-tarball/40/

File: binary-tarball-tests/ps/settings.py [hunk 1]
```diff
@@ -119,61 +120,60 @@
...
 
-if ps_version_major == '8.1':
-    ps_binaries = ps81_binaries
-    ps_executables = ps81_executables
-    ps_plugins = ps81_plugins
-    ps_functions = ps81_functions
-    ps_files = ps81_files
-    ps_symlinks = ps81_symlinks
-    ps_components = ps81_components
+if re.match(r'^8\.[1-3]$', ps_version_major):
```
Comment (eleo007) [inline binary-tarball-tests/ps/settings.py:169]: Why do we match only 1-3? We will have 8.4 etc in some time..

Comment (kaushikpuneet07) [reply binary-tarball-tests/ps/settings.py:169]: ok let me add till 9

Comment (kaushikpuneet07) [reply binary-tarball-tests/ps/settings.py]: fixed

**Review by eleo007 — APPROVED** [review]

**Pull request merged**